scripts/config-dhcp-client.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+S1="iface eth0 inet dhcp"
+
+sudo sed -i -e 's/\s*ip=[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*//' /boot/cmdline.txt
```

Comment:
Can you add a comment explaining what is going on ?
It's pretty hard to follow now, imagine in the future.
It's possible also to add an example of what the regex do ?